src/foundry/foundry.js/applications/formApplications/defaultTokenConfig.d.ts

```diff
@@ -30,11 +30,13 @@ declare global {
      * })
      * ```
      */
-    static get defaultOptions(): typeof FormApplication['defaultOptions'];
+    static override get defaultOptions(): typeof FormApplication['defaultOptions'];
```

Comment:
Did we decide to introduce using `override`? I think doing that would potentially affect users because they would require a newer typescript version. So far, we have only been using the `@override` tsdoc tag.